### ISSUE TITLE
"'Until' is not recognized as an internal or external command"

### ISSUE THREAD

**Issue by jaredcowing** [PeopleForBikes/brokenspoke-analyzer#484]
# Bug report

Me again, I'm trying again to run a local analysis using the Poetry virtual environment and I get the error: "'Until' is not recognized as an internal or external command"
I am unsure if this is an issue with my setup that I need to sort out on my own or an issue with the code, but thought I would check in case you had a sense for the nature of the issue.

## Current Behavior

Once in the virtual environment I initiate the analysis with this command:
`bna run-with compose usa "pittsfield" "massachusetts" 25`

The traceback starts with this:

```
 C:\Users\jec15\Documents\GitHub\brokenspoke-analyzer\brokenspoke_analyzer\c |
| li\run_with.py:55 in compose                                                |
|                                                                             |
|    52     database_url = "postgresql://postgres:postgres@localhost:5432/pos |
|    53     try:                                                              |
|    54         subprocess.run(["docker", "compose", "up", "-d"], check=True) |
| >  55         subprocess.run(                                               |
|    56             f"until pg_isready -d {database_url}; do sleep 5; done",  |
|    57             shell=True,                                               |
|    58             check=True,     
```

## Context

This error was produced in a Windows environment using Git Bash.

**Comment by rgreinho** [comment]
Ah yes, that's a problem on our end indeed! Thank you very much for reporting it. 

`until` is a bash keywork, mostly used on OSX or Linux, that's why it did not work for you. I will try to either move the logic to the `docker-compose.yml` file, maybe using a Healthcheck, or using a `docker run` command instead of the`until` one.

In the meantime, maybe using [gitbash](https://gitforwindows.org) or [cygwin](https://cygwin.com/install.html) would unblock you.

**Comment by rgreinho** [comment]
@jaredcowing, we believe we fixed the error you reported in PR #486. If you still encounter problems, don't hesitate to re-open this issue.

## Pittsfield, MA

Here are some extra information regarding the analysis you are trying to run that we believe may help you.

First, the FIPS CODE of Pittsfield, MA is `2553960`.  You can find the value in this [document](https://www2.census.gov/geo/docs/reference/codes2020/place/st25_ma_place2020.txt) from the US Census 2020. We describe the process to find FIPS CODEs in our [documentation](https://github.com/PeopleForBikes/brokenspoke-analyzer/blob/main/docs/source/about.md#where-to-find-the-fips-codes) (soon to be online, when we release v2.0.0).

Therefore the command to run would be:

```bash
bna -v run-with compose usa "pittsfield" "massachusetts" 2553960
```

Then, in case you were mostly interested by the final scores, here are the results we got with the data of today (Feb 3rd, 2023):

| score_id                                 | score_original | score_normalized | human_explanation                                                                             |
|:----------------------------------------:|:--------------:|:----------------:|:---------------------------------------------------------------------------------------------:|
| people                                   | 0.1897         | 18.9700          | "On average, census blocks in the neighborhood received this population score."               |
| opportunity_employment                   | 0.1541         | 15.4100          | "On average, census blocks in the neighborhood received this employment score."               |
| opportunity_k12_education                | 0.2227         | 22.2700          | "On average, census blocks in the neighborhood received this K12 schools score."              |
| opportunity_technical_vocational_college | 0.0751         | 7.5100           | "On average, census blocks in the neighborhood received this tech/vocational colleges score." |
| opportunity_higher_education             | 0.0000         | 0.0000           | "On average, census blocks in the neighborhood received this universities score."             |
| opportunity                              | 0.1742         | 17.4200          |                                                                                               |
| core_services_doctors                    | 0.1207         | 12.0700          | "On average, census blocks in the neighborhood received this doctors score."                  |
| core_services_dentists                   | 0.2099         | 20.9900          | "On average, census blocks in the neighborhood received this dentists score."                 |
| core_services_hospitals                  | 0.1530         | 15.3000          | "On average, census blocks in the neighborhood received this hospital score."                 |
| core_services_pharmacies                 | 0.1070         | 10.7000          | "On average, census blocks in the neighborhood received this pharmacies score."               |
| core_services_grocery                    | 0.0755         | 7.5500           | "On average, census blocks in the neighborhood received this grocery score."                  |
| core_services_social_services            | 0.1281         | 12.8100          | "On average, census blocks in the neighborhood received this social services score."          |
| core_services                            | 0.1245         | 12.4500          |                                                                                               |
| retail                                   | 0.2242         | 22.4200          | "On average, census blocks in the neighborhood received this retail score."                   |
| recreation_parks                         | 0.4301         | 43.0100          | "On average, census blocks in the neighborhood received this parks score."                    |
| recreation_trails                        | 0.0000         | 0.0000           | "On average, census blocks in the neighborhood received this trails score."                   |
| recreation_community_centers             | 0.1850         | 18.5000          | "On average, census blocks in the neighborhood received this community centers score."        |
| recreation                               | 0.3358         | 33.5800          |                                                                                               |
| transit                                  | 0.1129         | 11.2900          | "On average, census blocks in the neighborhood received this transit score."                  |
| overall_score                            | 0.1891         | 18.9100          |                                                                                               |
| population_total                         | 46957.0000     |                  | Total population of boundary                                                                  |
| total_miles_low_stress                   | 450.2497       | 450.2000         | Total low-stress miles                                                                        |
| total_miles_high_stress                  | 181.3376       | 181.3000         | Total high-stress miles                                                                       |

**Comment by jaredcowing** [comment]
Thank you! I'll give it a try. I appreciate the scores digest, we did an Openstreetmap edit-a-thon in late December and were curious how much closer that would bring our analysis & scores to the true state of things.
